.goreleaser.yml

```diff
@@ -26,7 +26,7 @@ checksum:
 
 release:
   prerelease: auto
-  name_template: "{{.Version}}"
+  name_template: "v{{.Version}}"
```

Comment:
Without the `v` prefix, the releases look like this: https://github.com/tendermint/tendermint/releases/tag/v0.34.24

This doesn't match the tags that we push, which include a `v` prefix.